cli/crates/cli/tests/compilation_error.rs

```diff
@@ -52,20 +81,31 @@ fn compilation_error() {
         "#,
     );
 
-    env.write_schema(COMPILATION_ERROR_RESOLVER_SCHEMA);
+    std::thread::sleep(std::time::Duration::from_secs_f64(0.5));
 
-    client.poll_endpoint_for_changes(30, 300);
+    // For now without watching before we investigate the issue.
+    env.grafbase_dev();
+
+    let client = env.create_client_with_longer_timeout().with_api_key();
```

Comment:
Maybe `client_with_options` and an options struct that lets you specify the timeout?

Reply:
Done